unlock_indicator.c

```diff
@@ -203,63 +283,54 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
-        cairo_set_font_size(ctx, 28.0);
-        switch (auth_state) {
-            case STATE_AUTH_VERIFY:
-                text = "verifying…";
-                break;
-            case STATE_AUTH_LOCK:
-                text = "locking…";
-                break;
-            case STATE_AUTH_WRONG:
-                text = "wrong!";
-                break;
-            case STATE_I3LOCK_LOCK_FAILED:
-                text = "lock failed!";
-                break;
-            default:
-                if (show_failed_attempts && failed_attempts > 0) {
-                    if (failed_attempts > 999) {
-                        text = "> 999";
-                    } else {
-                        snprintf(buf, sizeof(buf), "%d", failed_attempts);
-                        text = buf;
-                    }
-                    cairo_set_source_rgb(ctx, 1, 0, 0);
-                    cairo_set_font_size(ctx, 32.0);
-                }
-                break;
-        }
-
-        if (text) {
-            cairo_text_extents_t extents;
-            double x, y;
 
-            cairo_text_extents(ctx, text, &extents);
-            x = BUTTON_CENTER - ((extents.width / 2) + extents.x_bearing);
-            y = BUTTON_CENTER - ((extents.height / 2) + extents.y_bearing);
+        cairo_font_face_t *face = get_font_face();
 
-            cairo_move_to(ctx, x, y);
-            cairo_show_text(ctx, text);
-            cairo_close_path(ctx);
-        }
-
-        if (auth_state == STATE_AUTH_WRONG && (modifier_string != NULL)) {
-            cairo_text_extents_t extents;
-            double x, y;
+        if (!face) {
+            DEBUG("failed to get font face\n");
+        } else {
+            cairo_set_font_face(ctx, face);
+            cairo_set_font_size(ctx, 28.0);
+
+            switch (auth_state) {
+                case STATE_AUTH_VERIFY:
+                    text = "verifying…";
+                    break;
+                case STATE_AUTH_LOCK:
+                    text = "locking…";
+                    break;
+                case STATE_AUTH_WRONG:
+                    text = "wrong!";
+                    break;
+                case STATE_I3LOCK_LOCK_FAILED:
+                    text = "lock failed!";
+                    break;
+                default:
+                    if (show_failed_attempts && failed_attempts > 0) {
+                        if (failed_attempts > 999) {
+                            text = "> 999";
+                        } else {
+                            snprintf(buf, sizeof(buf), "%d", failed_attempts);
+                            text = buf;
+                        }
+                        cairo_set_source_rgb(ctx, 1, 0, 0);
+                        cairo_set_font_size(ctx, 32.0);
+                    }
+                    break;
+            }
 
-            cairo_set_font_size(ctx, 14.0);
+            if (text ) {
```

Comment:
Broken formatting

Reply:
The only time the font_size was set to 14px was [here](https://github.com/i3/i3lock/blob/master/unlock_indicator.c#L252). That got moved to [here](https://github.com/i3/i3lock/pull/168/files/0a03012d5af117e8631c23d66fb575daa0bbc5b0#diff-efe870ebf4bb57bf7e8e1316d4526cdaR327) in this patch.

Reply:
Sorry, that's not what I meant. I meant that the code formatting is wrong here because there's an extra space after `text`.

Reply:
Ahh, my bad. I didn't notice that, I'll fix that.